### ISSUE TITLE
palantir.utils.determine_multiscale_space returns 0 columns

### ISSUE THREAD

**Issue by csy99** [dpeerlab/Palantir#39]
(1469, 0) is the shape of data returned by the code. 
```python
[atac.txt](https://github.com/dpeerlab/Palantir/files/5059399/atac.txt)
// first load data
data = pd.read_csv("atac.txt")
dm_res = palantir.utils.run_diffusion_maps(data, n_components=5)
ms_data = palantir.utils.determine_multiscale_space(dm_res)  # something wrong with shape
```

**Comment by ManuSetty** [comment]
We will fix the bug. In the meantime, can you try setting the ```n_eigs``` parameter to the number of Eigen vectors which would be desirable.

**Comment by z5ouyang** [comment]
Having this again using above example. 
```
palantir 1.3.3
scanpy 1.10.2
anndata 0.11.0
```

**Comment by katosh** [comment]
Hi @z5ouyang,

Thank you for letting us know! I must have reverted Manus change at some point. I reintroduced the fix with 185780e0827c0d205777db25bdff4e2c3c15bdfd. So, you might be able to fix this by installing with
```bash
pip install --ignore-installed --no-deps 'git+https://github.com/dpeerlab/Palantir'
```
